tests/kw_db_test.sh

```diff
@@ -261,4 +261,121 @@ function test_select_from()
   assert_equals_helper 'Wrong output' "$LINENO" "$output" "$expected"
 }
 
+test_replace_into()
+{
+  local output
+  local expected
+  local ret
+
+  # invalid operations
+  output=$(replace_into table columns rows 'wrong/path/invalid_db.db')
+  ret="$?"
+  expected='Database does not exist'
+  assert_equals_helper 'Invalid db, error expected' "$LINENO" 2 "$ret"
+  assert_equals_helper 'Wrong error message' "$LINENO" "$expected" "$output"
+
+  output=$(replace_into '' columns rows)
+  ret="$?"
+  expected='Empty table or rows.'
+  assert_equals_helper 'Empty table, error expected' "$LINENO" 22 "$ret"
+  assert_equals_helper 'Wrong error message' "$LINENO" "$expected" "$output"
+
+  output=$(replace_into table columns '')
+  ret="$?"
+  expected='Empty table or rows.'
+  assert_equals_helper 'Empty rows, error expected' "$LINENO" 22 "$ret"
+  assert_equals_helper 'Wrong error message' "$LINENO" "$expected" "$output"
+
+  # valid operation with non-existent row
+  replace_into 'fake_table' '("name","attribute1","attribute2")' "('someName','someAtt1','someAtt2')"
+  ret="$?"
+  output=$(sqlite3 "$KW_DATA_DIR/kw.db" -batch "SELECT 'f'.'name' FROM 'fake_table' AS 'f' WHERE 'f'.'name'='someName' ;")
```

Comment:
https://kworkflow.org/content/codingstyle.html#string-concatenation

check the other parts of this test.